### ISSUE TITLE
Saving label vocab for serving

### ISSUE THREAD

**Issue by david-waterworth** [pytorch/text#1995]
## 🚀 Feature

It's not clear to me how to deal with text labels for classification. I built two transform pipelines, one is based on the example pipelines and applies a tokenizer, T.Truncate, T.VocabTransform etc and is applied to the text features. I also have text labels so I used a second pipeline based on T.LabelToIndex

None of the examples seem to cover this case, they only transform the features and use pre-numercised labels. When I came to save the model in order to use in a predict script / serving etc. its not clear what the best pattern is. I saved the T.LabelToIndex but this only maps from text -> integer. When you predict you need the inverse map integer -> text

What I ended up doing in my predict script was

    model = torch.load("model/model.pt")
    text_transform = torch.load("model/text_transform.pt")
    label_transform = torch.load("model/label_transform.pt")

    label_vocab = label_transform._label_vocab

    input = text_transform([text])
    output = model(input)

    label_transform.lookup_token(output.argmax(1).item())

I'm wondering if LabelToIndex should contain a method to perform the reverse transform (i.e. delegate to label_transform.lookup_token) - or is it a better to simply save the label_vocab rather than the label transform when training?

An end-to-end example containing predict with text labels would be useful.

(in general I really like the new torchdata / torchtext pipeline approach by the way)
